airflow/jobs/scheduler_job.py

```diff
@@ -1077,7 +1080,12 @@ def _enqueue_task_instances_with_queued_state(self, simple_dag_bag,
         :param simple_dag_bag: Should contains all of the task_instances' dags
         :type simple_dag_bag: airflow.utils.dag_processing.SimpleDagBag
         """
+
         TI = models.TaskInstance
+
+        def change_file_path(full_filepath):
+            return re.sub(DAGS_FOLDER, 'DAGS_FOLDER', full_filepath)
```

Comment:
This shouldn't be a regex (as the DAGS_FOLDER is a constant, not a regex) - `full_filepath.replace(DAGS_FOLDER, 'DAGS_FOLDER', 1)` should work.

Reply:
Right. Changes Done.